mars/services/storage/core.py

```diff
@@ -497,7 +497,15 @@ async def _setup_storage(self,
                              storage_config: Dict):
         backend = get_storage_backend(storage_backend)
         storage_config = storage_config or dict()
-        init_params, teardown_params = await backend.setup(**storage_config)
+
+        from ..cluster import ClusterAPI
+        try:
+            cluster_api = await ClusterAPI.create(self.address)
+            supervisor_address = (await cluster_api.get_supervisors())[0]
+        except mo.ActorNotExist:
+            supervisor_address = None
+        init_params, teardown_params = await backend.setup(
```

Comment:
How about adding special parameters for ray storage only? and the parameter could be `owner` instead of `supervisor_address`, `supervisor_address` is a bit weird for a storage lib.

Reply:
chanegd to pass owner to ray storage backend only